src/main/java/org/jenkinsci/plugins/gitclient/JGitAPIImpl.java

```diff
@@ -1178,6 +1178,10 @@ void format(RevCommit commit, @Nullable RevCommit parent, PrintWriter pw, Boolea
         }
     }
 
+    public CleanCommand clean_() {
+    	throw new UnsupportedOperationException("No JGit support!");
```

Comment:
Why not an implementation for JGit?  The [JGit CleanCommand](http://download.eclipse.org/jgit/site/4.6.0.201612231935-r/apidocs/org/eclipse/jgit/api/CleanCommand.html) seems like it could support path exclusion.